evaluator/src/test/kotlin/ProjectSourceRuleTest.kt

```diff
@@ -229,6 +229,6 @@ private fun createOrtResult(
                     )
                 )
             )
-        )
+        ).copy(scanners = mapOf(id to setOf("")))
```

Comment:
Please remove this `copy()` call and adjust `scannRunOf` to assign the scanner names internally, so that the return `ScannerRun` is consistent. (This way all callers of `scannerRunOf` get fixed.